src/types.ts

```diff
@@ -878,6 +880,9 @@ export interface DraxListProps<TItem> extends Omit<FlatListProps<TItem>, 'render
 	/** Callback handler for when a list item is moved within the list, reordering the list */
 	onItemReorder?: DraxListOnItemReorder<TItem>;
 
+	/** Callback handler for when list is scrolled */
+	onScrollProp?: (scrollEvent: NativeScrollEvent) => void;
```

Comment:
Is this leftover cruft from other branch work?

Reply:
ah this is another change I added this to our app as well so we could pass in a custom onScroll event. I could put this in another commit if you prefer.